backend/tests/daily/connectors/google_drive/test_user_1_oauth.py

```diff
@@ -0,0 +1,184 @@
+import time
+from collections.abc import Callable
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+from danswer.connectors.google_drive.connector import GoogleDriveConnector
+from danswer.connectors.models import Document
+from tests.daily.connectors.google_drive.consts_and_utils import ADMIN_FOLDER_3_FILE_IDS
+from tests.daily.connectors.google_drive.consts_and_utils import (
+    assert_retrieved_docs_match_expected,
+)
+from tests.daily.connectors.google_drive.consts_and_utils import FOLDER_1_1_FILE_IDS
+from tests.daily.connectors.google_drive.consts_and_utils import FOLDER_1_2_FILE_IDS
+from tests.daily.connectors.google_drive.consts_and_utils import FOLDER_1_FILE_IDS
+from tests.daily.connectors.google_drive.consts_and_utils import FOLDER_1_URL
+from tests.daily.connectors.google_drive.consts_and_utils import FOLDER_3_URL
+from tests.daily.connectors.google_drive.consts_and_utils import SHARED_DRIVE_1_FILE_IDS
+from tests.daily.connectors.google_drive.consts_and_utils import TEST_USER_1_EMAIL
+from tests.daily.connectors.google_drive.consts_and_utils import TEST_USER_1_FILE_IDS
+
+
+@patch(
+    "danswer.file_processing.extract_file_text.get_unstructured_api_key",
+    return_value=None,
+)
+def test_all(
+    mock_get_api_key: MagicMock,
+    google_drive_oauth_connector_factory: Callable[..., GoogleDriveConnector],
+) -> None:
+    print("\n\nRunning test_all")
+    connector = google_drive_oauth_connector_factory(
+        primary_admin_email=TEST_USER_1_EMAIL,
+        include_files_shared_with_me=True,
+        include_shared_drives=True,
+        include_my_drives=True,
+    )
+    retrieved_docs: list[Document] = []
+    for doc_batch in connector.poll_source(0, time.time()):
+        retrieved_docs.extend(doc_batch)
+
+    expected_file_ids = (
+        TEST_USER_1_FILE_IDS
+        + SHARED_DRIVE_1_FILE_IDS
+        + FOLDER_1_FILE_IDS
+        + FOLDER_1_1_FILE_IDS
+        + FOLDER_1_2_FILE_IDS
+        + ADMIN_FOLDER_3_FILE_IDS
+        + list(range(0, 2))
```

Comment:
what does this mean? And can we add a comment here as to why each of these should be included (this will be helpful for folks reading the rest of the test as well)?